soundbay/inference.py

```diff
@@ -134,8 +134,45 @@ def inference_to_file(
     output_file = output_path / filename
     concat_dataset.to_csv(index=False, path_or_buf=output_file)
 
+    # save raven file
+    threshold = 0.5
+    thresholdtext = int(threshold*10)
+    raven_filename = f"raven_annotations-{datetime.datetime.now().strftime('%Y-%m-%d_%H-%M-%S')}-{model_name}-{dataset_name}-thresh0{thresholdtext}.csv"
+    raven_output_file = output_path / raven_filename
+    raven_df = inference_csv_to_raven(results_df, test_dataset, threshold=threshold)
+    raven_df.to_csv(index=False, path_or_buf=raven_output_file, sep="\t")
```

Comment:
Does raven accept csv files as well? I thought it's only txt

Reply:
yup, it does. It was a marvelous discovery